plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/command/provider/FIDFeatureProvider.java

```diff
@@ -54,11 +54,11 @@ public synchronized SimpleFeature get(IProgressMonitor monitor, Object... params
                 monitor = new NullProgressMonitor();
             }
             try {
-                monitor.beginTask("Get Feature", 100);
+                monitor.beginTask("Get Feature", 100); //$NON-NLS-1$
```

Comment:
this and maybe other Strings are potentially a message that is visible in UI and should be externailzed to allow translations. This is not part of this pull request, but once marked as `$NON-NLS-1$` its hard to find the messages which needs to be exported in the future ;)

See Transifex project to invoke translation community : https://www.transifex.com/udig/udig/dashboard/